first-steps.mdx

```diff
@@ -66,80 +67,38 @@ Now we need to sign you up. Please, provide an email, so we can get in touch.
   />
 </Frame>
 
-## 5. View your test results
+## 4. Discover your test cases
 
-Access the Octomind app with the magic link. Welcome!
+Octomind will discover test cases on your page according to how your website looks.
```

Comment:
Sounds a bit weird to not talk about what happens to the test cases you selected during sign up, I'd add something like.
"Drafts of the test cases selected during sign-up can be found in the `drafts` section. Meanwhile, Octomind will discover ...."